website/project/views/node.py

```diff
@@ -1008,6 +1008,66 @@ def get_children(auth, node, **kwargs):
     return _render_nodes(nodes, auth)
 
 
+def node_child_tree(user, node_ids):
```

Comment:
It doesn't look like the existing method has everything I need (the privacy settings).  I could modify the existing method used by the WIKI in a future PR if you like, when we have time to test both the wiki and my code.